src/components/sidepanel.css

```
@@ -338,56 +291,23 @@ Border — Table
 border-top: 1px solid #CECECE;
 
 }
-.Frame21948 {
+/* .search-box {
```

do we need this "/*" here? if not then make it to this
```suggestion
 .search-box {
```